src/it/resources/com/google/checkstyle/test/chapter5naming/rule53camelcase/InputCamelCaseDefined.java

```diff
@@ -43,10 +43,9 @@ void XmlHttpRequest() {}
             void YouTubeImporter() {}
 
             void YoutubeImporter() {}
-    };
-}
+          };
 
-class AbbreviationsIncorrect {
+  class AbbreviationsIncorrect {
```

Comment:
Move this extra class inside the main class, reason mentioned here: https://github.com/checkstyle/checkstyle/pull/15239#discussion_r1676801130